db-init.js

```diff
@@ -8,26 +9,44 @@ if (NODE_ENV === 'production') throw new Error('The database should not be initi
 
 const client = new DynamoDB({ endpoint: 'http://localhost:8000' })
 
-;(async () => {
-  const params = {
-    AttributeDefinitions: [
-      {
-        AttributeName: 'id',
-        AttributeType: 'S'
-      }
-    ],
-    KeySchema: [
-      {
-        AttributeName: 'id',
-        KeyType: 'HASH'
-      }
-    ],
-    ProvisionedThroughput: {
-      ReadCapacityUnits: 5,
-      WriteCapacityUnits: 5
-    },
-    TableName: 'OnlineUsers'
-  }
-  const createTableCommandOutput = await client.createTable(params).catch((err) => console.log)
-  console.log(createTableCommandOutput)
-})()
+const onlineUsersParams = {
+  AttributeDefinitions: [
+    {
+      AttributeName: 'id',
+      AttributeType: 'S'
+    }
+  ],
+  KeySchema: [
+    {
+      AttributeName: 'id',
+      KeyType: 'HASH'
+    }
+  ],
+  ProvisionedThroughput: {
+    ReadCapacityUnits: 5,
+    WriteCapacityUnits: 5
+  },
+  TableName: tableOnlineUsers
+}
+await client.createTable(onlineUsersParams).catch((err) => console.log(err))
```

Comment:
When logging errors it makes sense to use `console.error` this, distinguishes logs from errors.

This is also done in `Players.js` on line 9 `updateOnlineUsersList().catch(e => console.log(e))`. Please fix it there as well.

If you notice it anywhere else, feel free to change it as well :)